main/acle.md

```diff
@@ -1509,24 +1509,34 @@ SVE language extensions:
 > [The __ARM_FEATURE_SVE_BITS macro](#the-__arm_feature_sve_bits-macro)
 > for details.
 
-**`__ARM_FEATURE_SVE_VECTOR_OPERATORS==1`**
+**`__ARM_FEATURE_SVE_VECTOR_OPERATORS==N`**
 
-> This indicates that applying the `arm_sve_vector_bits` attribute
+> N >= 1, this indicates that applying the `arm_sve_vector_bits` attribute
 > to an SVE vector type creates a type that supports the GNU vector
-> extensions. The state of this macro is only meaningful when
+> extensions. This condition is only meaningful when
 > `__ARM_FEATURE_SVE_BITS` is nonzero. See [`arm_sve_vector_bits` behavior
 > specific to vectors](#arm_sve_vector_bits-behavior-specific-to-vectors)
 > for details.
 
+> When N is 2, this indicates that the operators outlined in the GNU vector
+> extensions additionally work on sizeless SVE vector types like `svint32_t`.
+> The availability of operators on sizeless types is independent of
+> `__ARM_FEATURE_SVE_BITS`.
+
 **`__ARM_FEATURE_SVE_PREDICATE_OPERATORS==1`**
 
-> This indicates that applying the `arm_sve_vector_bits` attribute to
+> When N is 1, this indicates that applying the `arm_sve_vector_bits` attribute to
 > `svbool_t` creates a type that supports basic built-in vector operations.
 > The state of this macro is only meaningful when `__ARM_FEATURE_SVE_BITS`
 > is nonzero. See [`arm_sve_vector_bits` behavior specific to
 > predicates](#arm_sve_vector_bits-behavior-specific-to-predicates)
 > for details.
 
+> When N is 2, this indicates that the operators outlined in the clang vector
```

Comment:
Similarly:

> `N >= 2` indicates

I don't think “the clang vector extensions” is specific enough though.  We should either refer/point to the support for packed boolean types or perhaps just say:

> `N >= 2` indicates that the built-in vector operations described above additionally
> work on `svbool_t` itself.  The availability of operators on `svbool_t` is independent of
> `__ARM_SVE_FEATURE_BITS`.

Reply:
I've gone for the second one here as the packed boolean vector support in clang doesn't seem to be documented anywhere obvious, so I think it better not to refer to it.